server/server_test.go

```diff
@@ -4852,6 +4863,445 @@ func TestDistributedTracingEnabled(t *testing.T) {
 	}
 }
 
+func TestCertReloading(t *testing.T) {
```

Comment:
This is a big test, so I might be missing it, but are we making any assertions that the client CA pool file is respected; such as changes to the file being reflected in the client trust of incoming requests?

Reply:
I have added a new test for this in 63740e74b35f6be150477f54a967ffadbeb178e2. Thanks for the prompt!